components/event-publisher-proxy/pkg/sender/jetstream.go

```diff
@@ -12,9 +12,10 @@ import (
 	"go.uber.org/zap"
```

Comment:
the same with the naming here, can we rename this file, or rename the other files with the word `nats` in the name ?